### ISSUE TITLE
Establish some meta

### ISSUE THREAD

**Issue by sethp** [rustbox/discussion#11]
Let's get meta! As I understand it, the goals of this repo are:

* Practice clarifying our thinking by writing
* Low-barrier sharing of ideas with each other
* Longer-term discoverability than previous communication channels

and some non-goals:

* Instantaneous/synchronous feedback
* Polish for general audiences

Does that match your mental model @dougli1sqrd ?

Three rituals I have found helpful in the past:

a. Planned reviews of how we're doing (rather than spontaneous/ad-hoc)
b. Explicit coarse statements of values (that are intentionally ambiguous); I think `on-context.md` actually does a good job of establishing these
c. Very specific targeted actions (quite loosely held) 

I'm thinking we can close this issue when we add a `META.md` (or, as it's more commonly spelled, `README.md`) that outlines our shared expectations for how we revise & improve the "around game" loop here.

**Comment by dougli1sqrd** [comment]
Yeah I think this was to have a place to look at notes and things beyond signal, and to that end do some project planning or write some conclusions or thoughts about a project.

**Comment by sethp** [comment]
Yeah, we've got an embarrassment of communication channels: 

1. Signal
2. Commits
3. PRs
4. Issues
5. Discussions

So far, we've used all of them, and both of us are using them all differently. Which is what suggests to me it's worth going one level deeper: how do we want to do that?

**Comment by sethp** [comment]
Another thing that seems worth addressing here is that I'm not sure how best to communicate my current status, i.e 

* I think the next technical step on the router project is to purchase some FPGAs; that breaks down into 1) decide between the two PolarFire SBCs and/or 2) one of the RHS Research modules plus maybe a USB/thunderbolt adapter (they exist for the same form factor of SSDs, anyway)
* The next product step is probably finding some more comps & identifying who buys them; like, if we want to aim for the "network installer" persona that'll be different than the "homelab enthusiast" persona which is different from the "I have weird network problems I want to figure out" persona
* the thing I'm actually working on ( https://github.com/sethp/ucontext-coroutine ) has one or two more clearly defined objectives and then a few more vague objectives

As I count, that's ~10 or so "things" that are ready to be picked up. I would be shocked if they all seemed as clearly defined to you as they do to me, and I'm putting them here in an issue comment on an unrelated issue more as an example of what I'm curious about what to address, not necessarily to do the work for any of them right now. Does that make sense?
